Sources/TuistGraph/Models/Product.swift

```diff
@@ -139,7 +144,7 @@ public enum Product: String, CustomStringConvertible, CaseIterable, Codable {
         }
 
         if platform == .tvOS {
-            //            base.append(.tvExtension)
+            base.append(.tvTopShelfExtension)
```

Comment:
let's add commented out `tvIntensExtension` here as well. It will make it easier once we decide to implement that. Would you be so kind and raise an issue for that as well, so we can track this?